app/scripts/views/mixins/flow-events-mixin.js

```diff
@@ -24,8 +24,8 @@ define(function (require, exports, module) {
     },
 
     _clickFlowEventsLink (event) {
-      if (event && event.target) {
-        const flowEvent = $(event.target).data('flowEvent');
+      if (event && event.currentTarget) {
```

Comment:
Because app-store links contain images inside the link, I had to change this to `currentTarget` instead of `target` because `target` was always the image. `currentTarget` is the element the event is registered on.